src/css/base.scss

```diff
@@ -1,6 +1,56 @@
-/* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: pink;
+// COLORS
 
-body {
-  background-color: $primary-background;
```

Comment:
Snagged  9 variables and 5 mixins! The comments definitely help with the organization!